plugins/hotjar.ts

```diff
@@ -0,0 +1,19 @@
+declare global {
+  interface Window {
+    hj: any
+    _hjSettings: any
+  }
+}
+
+export default () => {
+  /* eslint-disable */
+  (function(h,o,t,j,a?: any,r?: any){
+    h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+    h._hjSettings={hjid:42920,hjsv:6};
```

Comment:
ID for new Hotjar's site for Qiskit.org is different.
You can find it here: https://insights.hotjar.com/sites/1586421/dashboard

So we need to change this ID as suggested below:
```suggestion
    h._hjSettings={hjid:1586421,hjsv:6};
```